ceph.spec.in

```diff
@@ -27,7 +27,7 @@
 # no gperftools/tcmalloc on s390(x)
 %bcond_with tcmalloc
 %endif
-%bcond_with lowmem_builder
+%bcond_without lowmem_builder
```

Comment:
lowmem_builder option activates this line:

```
RPM_OPT_FLAGS="$RPM_OPT_FLAGS --param ggc-min-expand=20 --param ggc-min-heapsize=32768"
```

which (long ago) we found fixed an OOM situation in the build service. If you like, we could try dropping it entirely, but the OBS/IBS builds have been so stable with this option (which has been enabled via prjconf in all OBS/IBS builds ever since it was implemented) that I am a little reluctant to touch it.